test/openclon12test.cpp

```diff
@@ -201,10 +202,90 @@ TEST(OpenCLOn12, Printf)
     cl::Program program(context, kernel_source, true /*build*/);
     cl::Kernel kernel(program, "test_printf");
 
-    queue.enqueueTask(kernel);
+    queue.enqueueNDRangeKernel(kernel, cl::NullRange, cl::NDRange(1));
     queue.finish();
 }
 
+TEST(OpenCLOn12, SPIRV)
+{
+    static const unsigned char spirv[] = {
+0x03, 0x02, 0x23, 0x07, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x07, 0x00, 0x22, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x11,

```

Comment:
It'd be helpful to comment on the origin of this magic test data

Reply:
Agreed, thanks.